contract/vm.go

```diff
@@ -783,8 +794,8 @@ func (ce *executor) close() {
 	}
 }
 
-func (ce *executor) getRemainingGas() {
-	ce.ctx.getRemainingGas(ce.L)
+func (ce *executor) refreshRemainingGas() {
+	ce.ctx.refreshRemainingGas(ce.L)
```

Comment:
You can rename the one from the executor, but keep the one of vmContext unchanged because it forms a pair of functions with similar functionality and name